Source/Mixin.MDKProjectProperties/MDKProjectOptions.cs

```diff
@@ -481,4 +499,18 @@ public bool IsIgnoredFilePath(string filePath)
             return false;
         }
     }
+    /// <summary> 
+    /// Describes script minification level. 
+    /// <para>Stored as enum item name, not as numeric value, in case a reordering/extension is needed.</para>
+    /// <para>However, due to complications with WPF, values of items in combobox should match these values.</para>
```

Comment:
Note To Self: Remove this comment section as it is only relevant to the _current_ implementation of the combo box and won't hold true after the rewrite.